### ISSUE TITLE
MasterSlaveConnection and configuration of default_table_options 

### ISSUE THREAD

**Issue by Spawnrad** [doctrine/DoctrineBundle#900]
Hi,

When i use MasterSlave configuration on doctrine the parameter default_table_options is not taken.
But once i remove the slave configuration it work and my tables is create with my charset configuration.

Here my doctrine configuration : 

```
doctrine:
    dbal:
        driver:   pdo_mysql
        host:     "%env(DATABASE_MASTER_HOST)%"
        dbname:   "%env(DATABASE_DBNAME)%"
        user:     "%env(DATABASE_USER)%"
        password: "%env(DATABASE_PASSWORD)%"
        slaves:
            slave1:
                host:     "%env(DATABASE_SLAVE_HOST)%"
                dbname:   "%env(DATABASE_DBNAME)%"
                user:     "%env(DATABASE_USER)%"
                password: "%env(DATABASE_PASSWORD)%"
        charset: utf8mb4
        default_table_options:
            charset: utf8mb4
            collate: utf8mb4_unicode_ci  
```
Its looks like that the defaultTableOptions have no data when i use MasterSlave configuration : 

```
array:6 [
  "driver" => "pdo_mysql"
  "slaves" => array:1 [
    "slave1" => array:6 [
      "host" => "db:3306"
      "dbname" => "databasename"
      "user" => "root"
      "password" => "root"
      "port" => null
      "driver" => "pdo_mysql"
    ]
  ]
  "driverOptions" => []
  "master" => array:8 [
    "host" => "db:3306"
    "dbname" => "databasename"
    "user" => "root"
    "password" => "root"
    "charset" => "utf8mb4"
    "port" => null
    "defaultTableOptions" => array:2 [
      "charset" => "utf8mb4"
      "collate" => "utf8mb4_unicode_ci"
    ]
    "driver" => "pdo_mysql"
  ]
  "wrapperClass" => "Doctrine\DBAL\Connections\MasterSlaveConnection"
  "defaultTableOptions" => []
]
```

Once i remove the slave for example i have that result and it work very well : 

```
array:9 [
  "driver" => "pdo_mysql"
  "host" => "db:3306"
  "dbname" => "databasename"
  "user" => "root"
  "password" => "root"
  "charset" => "utf8mb4"
  "port" => null
  "driverOptions" => []
  "defaultTableOptions" => array:2 [
    "charset" => "utf8mb4"
    "collate" => "utf8mb4_unicode_ci"
  ]
]
```

Thank you

**Comment by Spawnrad** [comment]
In the MasterSlaveConnection if $params['master']['defaultTableOptions'] exist we need to : 

$params['defaultTableOptions'] = $params['master']['defaultTableOptions']

If i do that it work very well.

**Comment by Spawnrad** [comment]
any news please?

**Comment by alcaeus** [comment]
Fixed in #915.

**Comment by Spawnrad** [comment]
Thank you @alcaeus
